docs/userguide/surrogates.md

```diff
@@ -1,18 +1,49 @@
 # Surrogates
 
-Surrogate models are used to model and estimate the unknown objective function of the DoE campaign. BayBE offers a diverse array of surrogate models, while also allowing for the utilization of custom models. All surrogate models are based upon the general [`Surrogate`](baybe.surrogates.base.Surrogate) class. Some models even support transfer learning, as indicated by the `supports_transfer_learning` attribute.
+Surrogate models are used to model and estimate the unknown objective function of the
+DoE campaign. BayBE offers a diverse array of surrogate models, while also allowing for
+the utilization of custom models. All surrogate models are based upon the general
+[`Surrogate`](baybe.surrogates.base.Surrogate) class. Some models even support transfer
+learning, as indicated by the `supports_transfer_learning` attribute.
 
 ## Available models
 
-BayBE provides a comprehensive selection of surrogate models, empowering you to choose the most suitable option for your specific needs. The following surrogate models are available within BayBE:
+BayBE provides a comprehensive selection of surrogate models, empowering you to choose
+the most suitable option for your specific needs. The following surrogate models are
+available within BayBE:
 
 * [`GaussianProcessSurrogate`](baybe.surrogates.gaussian_process.core.GaussianProcessSurrogate)
 * [`BayesianLinearSurrogate`](baybe.surrogates.linear.BayesianLinearSurrogate)
 * [`MeanPredictionSurrogate`](baybe.surrogates.naive.MeanPredictionSurrogate)
 * [`NGBoostSurrogate`](baybe.surrogates.ngboost.NGBoostSurrogate)
 * [`RandomForestSurrogate`](baybe.surrogates.random_forest.RandomForestSurrogate)
 
+## Extracting the Model for Advanced Study
 
-## Using custom models
+In principle, the surrogate model does not need to be a persistent object during
+Bayesian optimization since each iteration performs a new fit anyway. However, for
+advanced study, such as investigating the posterior predictions, acquisition functions
+or feature importance, it can be useful to diretly extract the current surrogate model.
 
-BayBE goes one step further by allowing you to incorporate custom models based on the ONNX architecture. Note however that these cannot be retrained.  For a detailed explanation on using custom models, refer to the comprehensive examples provided in the corresponding [example folder](./../../examples/Custom_Surrogates/Custom_Surrogates).
\ No newline at end of file
+For this, BayBE provides the ``get_surrogate`` method, which is available for the
+[``Campaign``](baybe.campaign.Campaign.get_surrogate) or for 
+[recommenders](baybe.recommenders.pure.bayesian.base.BayesianRecommender.get_surrogate).
+Below an example of how to utilize this in conjunction with the popular SHAP package:
```

Comment:
The note that is in the code (or a variant of it) should also be included here as people who only read the user guide might not read the code.

Reply:
makes sense, added